nextflow.config

```diff
@@ -205,10 +206,13 @@ manifest {
     homePage        = 'https://github.com/nf-core/chipseq'
```

Comment:
Add our names in the author field above :)

Reply:
Done in 9eb3c17951b83774e42889b4797ef28ae203945f :boom: 
BTW, your name is not in the manifest of the rnaseq pipeline :scream: